test/cctest/cctest.h

```diff
@@ -74,9 +74,11 @@ static inline void CheckEqualsHelper(const char* file, int line,
                                      const char* expected,
                                      const char* value_source,
                                      const char* value) {
-  if ((expected == NULL && value != NULL) ||
-      (expected != NULL && value == NULL) ||
-      (expected != NULL && value != NULL && strcmp(expected, value) != 0)) {
+  if (expected == NULL || value == NULL) {
+      abort();
+    }
```

Comment:
The indentation looks off to me.

This is not doing the same thing: if the expected and the value should be NULL you would now abort.